skills/pixiv-cli/references/discover.md

```
@@ -57,6 +57,40 @@ pixiv search "初音ミク" --type artwork --limit 10 --json
 - Artwork JSON/text include the stable page URL
   `https://www.pixiv.net/artworks/{id}` as the first field/line.
 
+## Reverse-search an image
+
+Use the same `search` command with a local regular file or an HTTP(S) image URL:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)